### PR TITLE
SSL cert ordering article

### PR DESCRIPTION
**Pull request by wcpines**
Initial draft of content.  @fancyremarker for review.

### CODE REVIEW

**Comment by wcpines** [comment]
should fix #164

**Comment by fancyremarker** [comment]
Looks like this PR got messed up with unintended commits. Let's close and @gib will help to rebase. [edited]

**Comment by wcpines** [comment]
Thought I rebased correctly :(

**Comment by gib** [comment]
Thanks for doing this one @wcpines!

File: source/topics/paas/how-to-order-certs.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+This error is the result of either not including the necessary intermediate certs, or due to improper formatting.
```
Comment (fancyremarker) [inline source/topics/paas/how-to-order-certs.md:1]: The content of this article is very strong. Minor copy edit for syntax: remove "due to".

**Comment by gib** [comment]
Closing. @wcpines, let me know if/when I can help with a rebase

**Pull request closed without merging**